src/search.js

```diff
@@ -0,0 +1,8 @@
+import { mentors } from "./mentors.json";
+
+const fieldSearch = ({ field, value }) =>
+  mentors
+    .map((mentor, index) => index)
+    .filter((index) => mentors[index][field] === value);
```

Comment:
This can probably be simplified to `.map((mentor) => mentor[field] === value)`.

Reply:
The intention was to get a list of indexes at the end without changing the `index` :P have to say this index business with the `mentors` is pretty annoying and I should get to generating an unique ID for each

Reply:
Oh right, scratch that :)